crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -596,13 +640,52 @@ where
 
                 self.top_level_node = top_level_node;
 
+                let args = once(name.as_arg()).chain(once(props_obj.as_arg()));
+                let args = if self.development {
+                    // set undefined literal to key if key is None
+                    let key = match key {
+                        Some(key) => key,
+                        None => ExprOrSpread {
+                            spread: None,
+                            expr: Box::new(build_undefined_literal_value()),
```

Comment:
Please use `undefined(DUMMY_SP)` instead.

Reply:
I did not notice `undefined` util module :bow:

:+1: 0ac5ded